CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ _None._
 
 ### Breaking Changes
 
-_None._
+- Add a new `unacceptableStatusCode` error case to `WordPressAPIError`. [#668]
```

Comment:
How timely. I left a comment on a PR recently that introduced breaking changes, https://github.com/wordpress-mobile/WordPress-iOS/pull/22320#issuecomment-1882125134, asking to make them non-breaking with a reminder to update them when we have enough of a corpus of breaking changes to warrant the "effort" of updating WordPressAuthenticator as well. 

This seems to be the case now 😄

Reply:
Followed up here: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/678